charts/backstage/values.yaml

```diff
@@ -244,8 +245,8 @@ upstream:
     enabled: true
     postgresqlDataDir: /var/lib/pgsql/data/userdata
     image:
-      registry: quay.io
-      repository: fedora/postgresql-15
+      registry: registry.redhat.io
```

Comment:
This will likely be an issue when testing the Chart on GH, since reistry.redhat.io requires authentication. I'd suggest keeping the image from quay.io in the default values file.